join_github_app/templates/pages/digital-justice-user.html

```diff
@@ -0,0 +1,48 @@
+{% extends "components/base.html" %}
+
+{% block pageTitle %}
+Using SSO for GitHub Access
+{% endblock %}
+
+{% block beforeContent %}
+{{ govukBackLink ({
+'text': 'Back',
+'href': '/select-organisations'
+}) }}
+{% endblock %}
+
+{% block content %}
+<h1 class="govuk-heading-l">Using Single Sign-On Access</h1>
+
+<p class="govuk-body">
+  Your email address <strong>{{email}}</strong> is registered for SSO access to the GitHub organisation. Single Sign-On
+  (SSO) is a convenient
+  method allowing you to access multiple services with one set of login credentials.
+</p>
+<p class="govuk-body">
+  By clicking the button below for the chosen organisation, you'll be directed to a secure login
```

Comment:
```suggestion
  Click the button below for the chosen organisation to open a secure login
```
Best to avoid auxiliary verbs (will) to keep the language simple, and preferably present tense, active voice.